packages/@aws-cdk/cx-api/lib/versioning.ts

```diff
@@ -75,7 +75,7 @@ export function upgradeAssemblyManifest(manifest: AssemblyManifest): AssemblyMan
     // Backwards compatible changes to ContainerImageAssetMetadataEntry:
     // * Make `imageNameParameter` optional (new apps do not require it anymore because container images go to a well-known repository)
     // * Add optional `imageTag` to allow apps to specify exactly where to store the image (required if `imageNameParameter` is not defined)
-    manifest = justUpgradeVersion(manifest, '1.21.0');
+    manifest = justUpgradeVersion(manifest, '1.23.0');
```

Comment:
Shouldn't this also include a section upgrading the version from `1.21.0` to `1.23.0`?

We will be publishing Cloud Assemblies with version `1.21.0` with a few versions of the framework (`1.22.0` at least).

Reply:
You are so right. Good catch! Fixed 👍

Reply:
+1